src/main/scala/fs2/pull1.scala

```diff
@@ -51,6 +51,17 @@ private[fs2] trait pull1 {
   def await1Option[F[_],I]: Handle[F,I] => Pull[F,Nothing,Option[Step[I,Handle[F,I]]]] =
     h => h.await1.map(Some(_)) or Pull.pure(None)
 
+  def changing[F[_],I](eqf:(I,I) => Boolean):Handle[F,I] => Pull[F,I,I] = {
```

Comment:
I wouldn't put this in `pull1`, since it doesn't return a useful result, it just outputs values so it really belongs as a helper function in `process1`. Also, I would implement using `zipWithLast` and `collect`. It will be more efficient and preserves chunkiness.